### ISSUE TITLE
Allow controlling the html report verbose logging separately from fastlane verbose flag

### ISSUE THREAD

**Issue by lilyball** [lyndsey-ferguson/fastlane-plugin-test_center#299]
#### Motivation Behind Feature
I run `fastlane` with the `--verbose` flag because it's really useful when diagnosing failures, especially with long-running jobs that I don't want to have to repeat locally.

The downside is `multi_scan` dumps a ton of output under verbose when it's collating HTML reports, as it prints the entire raw HTML contents of the report. I'd really like a way to disable printing of these HTML reports while keeping verbose on for everything else.

#### Feature Description
Give me some flag I can set in my `Fastfile` to enable/disable verbose logging of html report collating.

**Comment by lyndsey-ferguson** [comment]
@lilyball Can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane like so:

```
COLLATE_HTML_REPORTS_VERBOSITY=0 COLLATE_JUNIT_REPORTS_VERBOSITY=0 bundle exec fastlane <your-lane> --verbose
```


Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-299-allow-html-junit-verbosity-to-be-turned-off"
```

If there are still problems, please let me know and attach the console output as a text file to this issue (makes it easier for me to review).  If it works, please let me know.

**Comment by lyndsey-ferguson** [comment]
Released in [v3.14.8](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.14.8)
